### DISCUSSION TITLE
Native form validation in vue 3

### DISCUSSION THREAD

**Discussion by yashwp** [vuejs/rfcs#658] [General]
Hi,

I think similar to Angular Framework vue 3 team should also look into, native form validation to make life easier.

There is no framework way of Vue where we can check if the field is `.dirty` or `.pristine` or `.touched` to trigger validations.

Is it in the pipeline by any chance?

**Comment by nickmessing** [comment]
This doesn't feel like something vue should do. Angular is a huge framework while vue keeps the scope of responsibility significantly smaller. There are multiple libraries that do this, I don't think bloating the core rendering library with something like this is viable.

**Comment by dragonsdenexotech** [comment]
Me too want this feature! I found something similar. I wish there was a way to integrate this with any vue project; without the need to integrate the whole inkline library.
https://www.inkline.io/docs/forms/validation
